### DISCUSSION TITLE
Airflow Webserver, Triggerer and Scheduler pods often stucked in init state whenever there is a restart

### DISCUSSION THREAD

**Discussion by dammyabioye1** [apache/airflow#21582] [General]
### Apache Airflow version

2.2.3 (latest released)

### What happened

Airflow running on Kubernetes Executor refuses to restart webserver, truggerer and scheduler pods succesfully.

### What you expected to happen

The pods should restart successfully without any manual intervention.

### Operating System

Debian GNU/Linux version 10

### Versions of Apache Airflow Providers

apache-airflow-providers-apache-spark==2.0.3
apache-airflow-providers-celery==2.1.0
apache-airflow-providers-cncf-kubernetes==2.2.0

### Deployment

Official Apache Airflow Helm Chart

### Deployment details

Restart all the resources in the namespace or perform helm install upgrage.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
There are no logs, details, no reproduction scenario. This is at most a discussion. Please provide details about your deployment and the exact steps you do to see the behaviour if you experience it. It might be a problem with how your configuration and deployment is, so exact information on your specific configuration might be needed to help you, but you need to provide some logs/ information on what's happening, where more information about signals, and sequence of things happening will be visible. This is not something that we observe and our tests are testing both installation and upgrade scenarios using the chart so if anything - this is something specific for you.

Please provide more information if you need some help here.
